xcodes, xcodes-runtimes: add page (#10718)
tldr-pages/tldr: pages/osx/xcodes.md

# xcodes

> > Download, install and manage multiple Xcode versions.
> > More information: <https://github.com/xcodesorg/xcodes>.

Reviewer: ```suggestion
> Download, install and manage multiple Xcode versions.
> See also: `xcodes runtimes`.
> More information: <https://github.com/xcodesorg/xcodes>.
```

- List all installed Xcode versions:

`xcodes installed`

- List all available Xcode versions:

`xcodes list`

> - Select an Xcode version by specifying a version number or path:
> 
> `xcodes select {{xcode-version|/path/to/Xcode.app}}`

Reviewer: ```suggestion
- Select an Xcode version by specifying a version number or a path:

`xcodes select {{xcode-version|path/to/Xcode.app}}`
```

- Download and install a specific Xcode version:

`xcodes install {{xcode-version}}`

> - Install the latest 'release' Xcode version and select it:

Reviewer: ```suggestion
- Install the latest Xcode release and select it:
```

`xcodes install --latest --select`

- Download a specific Xcode version archive to a given directory without installing it:

> `xcodes download {{xcode-version}} --directory {{path/to/download/directory}}`

Reviewer: ```suggestion
`xcodes download {{xcode-version}} --directory {{path/to/directory}}`
```
